### DISCUSSION TITLE
About known AUR packages

### DISCUSSION THREAD

**Discussion by oguzkaganeren** [oguzkaganeren/manjaro-starter#364] [Ideas]
I realized many known applications are at AUR (Spotify, android-studio, wps-office, vscodium-bin, woeusb, zoom, etc.). So, I think we can add an AUR package screen or tab (not sure yet). I guess, they can be installed by `pamac build` (if not, doesn't matter, I can find a solution). What do you think? If you think it would be good, can you share your AUR packages?

**Comment by oguzkaganeren** [comment]
@yochananmarqos @Tuxman2 @dai5m5iq

**Comment by yochananmarqos** [comment]
I don't think this is a good idea. The AUR is not enabled by default in Pamac for a reason. It's neither officially supported by Arch nor Manjaro. 

Users will never learn to use the AUR properly if their first introduction is a package manager that does it for them. AUR helpers are meant to be a convenience to do something that can be tedious to do manually. One should learn how to do it manually before using an AUR helper.

Also, we are considering splitting out AUR support into a separate plugin like Flatpak and Snap are done.
